### DISCUSSION TITLE
Please remove Prism v9000.0.1

### DISCUSSION THREAD

**Discussion by RunDevelopment** [cdnjs/cdnjs#14145] [General]
Hello!

I am one of [Prism](https://github.com/PrismJS/prism)'s maintainers (npm [`prismjs`](https://www.npmjs.com/package/prismjs)) and we recently noticed that [version 9000.0.1 still available via cdnjs.com](https://cdnjs.com/libraries/prism). This is a version we published by mistake and have removed from npm a while ago. Due to the extremely high version number, it is also the version cdnjs picks by default, which [causes problems](https://github.com/PrismJS/prism/issues/819#issuecomment-1182299429).

Would it be possible to remove v9000.0.1? 

For more information about version 9000.0.1, see https://github.com/PrismJS/prism/issues/819.

**Comment by MattIPv4** [comment]
👀 We have an open issue on the cdnjs/packages repository about this: https://github.com/cdnjs/packages/issues/926

We will not be able to unpublish the version as that will be breaking for anyone consuming it on a webpage, but we may be able to change the default version being recommended.

**Reply by RunDevelopment** [reply]
> as that will be breaking for anyone consuming it on a webpage,

I would argue that anyone relying on a mistake should expect their webpage to break. This version is basically a bug. But I can also understand that you want cdnjs to provide certain guarantees.

> we may be able to change the default version being recommended.

That would be great. v9000.0.1 is years old and contains multiple security vulnerabilities. So it being the default version is actively harmful to users.

**Reply by MattIPv4** [reply]
@RunDevelopment Would you mind posting over in the linked cdnjs/packages issue with your support for removing this from being the default version?
